test/system/030-run.bats

```diff
@@ -736,4 +736,11 @@ EOF
     is "$output" "$random_1" "output matches STDIN"
 }
 
+# rhbz#1902979 : podman run fails to update /etc/hosts when --uidmap is provided
+@test "podman run update /etc/hosts" {
+    HOSTS=$(random_string 25)
+    run_podman run --uidmap 0:10001:10002 --rm --hostname $HOSTS $IMAGE grep $HOSTS /etc/hosts
+    is "${lines[0]}" ".*$HOSTS.*"
```

Comment:
This check is unnecessary: if the `grep` above were to fail, `run_podman` would issue a failure.